pfio/FileMetadata.F90

```diff
@@ -101,6 +102,14 @@ function get_dimensions(this) result(dimensions)
 
    end function get_dimensions
 
+   function get_global(this) result(global)
+      type (Variable), pointer :: global
+      class (FileMetadata), target, intent(in) :: this
+
+      global => this%global
+
+   end function get_global
+
```

Comment:
"Global" is a bit vague here.  Could be global attribute or global variable.   I suggest (below) a more specific name.

```suggestion
   function get_global_var(this) result(global)
      type (Variable), pointer :: global
      class (FileMetadata), target, intent(in) :: this

      global => this%global

   end function get_global_var

```

Reply:
Could you also change the `global` component of the derived type to `global_var` as part of your change?   I can't "suggest" it here because you don't have edits on that line.  Consistency ...